QKSMS/src/main/java/com/moez/QKSMS/ui/view/ComposeView.java

```diff
@@ -753,7 +755,40 @@ public void saveDraft() {
 
```

Comment:
`mPrefs.edit().putString(QKPreference.COMPOSE_DRAFT.getKey(), draft).apply();`

This line will save a draft for no recipient. It doesn't work using the standard draft saving method (since it requires it to be a part of a conversation)